auto/data/str_quote.functional_test.in

```diff
@@ -2,4 +2,12 @@ echo "hello world"
 echo 'hello world'
 echo "hello world" 'hi ni'
 echo hello world
-echo "j's'f"
\ No newline at end of file
+echo "j's'f"
+echo $haha
+echo $EXIST_PATH_ENV
+echo 123$EXIST_PATH_ENV
+echo $123EXIST_PATH_ENV
+echo $EXIST_PATH_ENV.123
+echo $EXIST_PATH_ENV123
+echo $123EXIST_PATH_ENV.123
+echo 123$EXIST_PATH_ENV.123
```

Comment:
Can you also add case for quotation mark? For example: `echo '$EXIST_PATH_ENV'` and `echo "$EXIST_PATH_ENV"`